packages/shared/src/contexts/BootProvider.tsx

```diff
@@ -91,7 +91,7 @@ export const BootDataProvider = ({
 
   useRefreshToken(bootData?.accessToken, refetch);
   const { user, updateUser, loadedFromCache } = useCacheUser(bootData?.user);
-  const alertContext = useAlertContext(null);
+  const alertContext = useAlertContext();
```

Comment:
Shouldn't we provide here `bootData?.alerts`?

Reply:
Yep 100% 
Didn't realise we had this available already, modified now